app/src/main/kotlin/com/mapzen/erasermap/presenter/MainPresenterImpl.kt

```diff
@@ -842,18 +854,23 @@ open class MainPresenterImpl(val mapzenLocation: MapzenLocation, val bus: Bus,
     return rawFeature
   }
 
-  override fun onFeaturePicked(properties: Map<String, String>, x: Float, y: Float) {
+  override fun onFeaturePicked(properties: Map<String, String>?, coordinates: LngLat?, x: Float,
```

Comment:
Change `coordinates` to `coords` to be consistent with parameter name in `MainPresenter` interface?